lib/everypolitician/popolo/organization.rb

```diff
@@ -4,6 +4,34 @@ class Organization < Entity
       def wikidata
         identifier('wikidata')
       end
+
+      def classification
+        document.fetch(:classification, nil)
+      end
+
+      def identifiers
+        document.fetch(:identifiers, [])
+      end
+
+      def image
+        document.fetch(:image, nil)
+      end
+
+      def links
+        document.fetch(:links, [])
+      end
+
+      def name
+        document.fetch(:name, nil)
+      end
+
+      def other_names
+        document.fetch(:other_names, [])
+      end
+
+      def seats
+        document.fetch(:seats, nil)
```

Comment:
I suspect it's worth having a "TODO" here noting that this is a temporary measure until we split out separate Legislature and Party classes.